rosdep/python.yaml

```diff
@@ -2632,6 +2632,19 @@ python-mechanize:
   gentoo: [dev-python/mechanize]
   nixos: [pythonPackages.mechanize]
   ubuntu: [python-mechanize]
+python3-mediapipe-pip:
```

Comment:
you need to use alpha order to make the tests pass.
You can use rosdistro_reformat, which is available after installing python3-rosdistro (example [here](http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation#Steps_to_have_your_repository_added_to_our_indexer))

Reply:
Rookie mistake... Its sorted now.

Reply:
no problem, thanks for updating!